Features/DriverJS/Sahi.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\PlatformUIBundle\Features\DriverJS;
+
+/**
+ * Implementation of Sahi Driver
+ *
+ */
+class Sahi implements DriverJSInterface
+{
+    /**
+     * Adapts javascript to run on Sahi webDriver
+     *
+     * @param string    $code javacript code to adapt
+     *
+     * @return string   adapted javascript code
+     */
+    public function wrappedFunction( $code )
+    {
+        $jsCode = "(function(){
```

Comment:
actually, if we ever need to inject some custom JavaScript code, we should always put it inside such a function. So to me, the whole `DriverJSInterface` thing seems useless.